crates/yttrium/src/chain_abstraction/api/route.rs

```diff
@@ -18,22 +16,25 @@ pub struct RouteRequest {
 }
 
 #[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+#[cfg_attr(feature = "uniffi", derive(uniffi_macros::Record))]
 #[serde(rename_all = "camelCase")]
 pub struct Metadata {
     pub funding_from: Vec<FundingMetadata>,
-    pub check_in: usize,
+    pub check_in: u32,
```

Comment:
I feel like we should use `u64` instead here. That's what [`Duration::from_millis()`](https://doc.rust-lang.org/std/time/struct.Duration.html#method.from_millis) accepts

Reply:
Actually we could go to `Duration` soon as uniffi supports that. We just have to make sure serde does too